### PR TITLE
design updates for v4

### PR DESCRIPTION
**Pull request by vanhue**
- improved header to expandable header componenet
- improved typography of headers
- updated colors

### CODE REVIEW

**Comment by vanhue** [comment]
Just fixed. @cmgustavo

**Comment by cmgustavo** [comment]
Another issue when loading paypro invoice in wallets page

Screen Shot 2020-02-04 at 12 49 42

File: src/pages/home/home.ts [hunk 1]
```diff
@@ -78,7 +78,7 @@ export class HomePage {
       app: 'bitpay',
       linkText: 'View Directory',
       link: 'https://bitpay.com/directory/?hideGiftCards=true',
-      imgSrc: 'assets/img/gumball-3.svg',
+      imgSrc: 'assets/img/icon-merch-dir.svg',
```
Comment (cmgustavo) [inline src/pages/home/home.ts]: Missing icon. Probably it's a typo and you wanted use this new one: `mcc-icons/merch-dir.svg`

**Comment by cmgustavo** [comment]
In order to simplify review of the code and number of lines, might you try to reuse this component: `wide-header-page`. That component help to write less lines of code. It's widely used in the app.

**Comment by Gamboster** [comment]
Hey, Good job so far! I'm taking a look now.
Could you avoid formatting the html lines? Throughout the copay code we use the html tags in a single line.

**Comment by Gamboster** [comment]
WhatsApp

A detail: In wallets view, the loading spinner appears below the title, as seen in the photo.
In the Home view it works correctly.

**Comment by cmgustavo** [comment]
Needs rebase

**Comment by vanhue** [comment]
Updated

**Comment by cmgustavo** [comment]
LGTM. Great work!

**Pull request closed without merging**